pubspec.lock

```diff
@@ -6,7 +6,7 @@ packages:
       name: analyzer
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.30.0"
+    version: "0.30.0+4"
```

Comment:
any particular reason for the upgrades?

Reply:
get the latest markdown, gets rid of which/when – which aren't used.